charon-ml/src/GAstOfJson.ml

```diff
@@ -912,6 +912,9 @@ and literal_of_json (js : json) : (literal, string) result =
     | `Assoc [ ("Scalar", scalar) ] ->
         let* scalar = scalar_value_of_json scalar in
         Ok (VScalar scalar)
+    | `Assoc [ ("Float", float) ] ->
+        let* float = float_value_of_json float in
```

Comment:
This won't work: you serialized the float value to a string, this expects a json float